frontend/www/js/omegaup/components/problem/FilterAuthors.vue

```diff
@@ -0,0 +1,41 @@
+<template>
+  <div class="card">
+    <div class="card-body">
+      <h3>{{ T.problemCollectionAuthors }}</h3>
+      <div v-for="(author, index) in authors" :key="index" class="form-check">
```

Comment:
```suggestion
      <div v-for="(author, index) in authors" :key="author" class="form-check">
```

normalmente se evita usar el índice como clave porque la intención es que si el arreglo cambia (digamos, se reemplaza un elemento por otro), Vue sepa qué elemento quitar y qué poner en su lugar.